### PR TITLE
fix: use TARGETARCH to select the git-credential-github-app artifact

### PR DESCRIPTION
**Pull request by Meroje**
## What is the problem I am trying to address?

Using the new github-app credential feature on arm isn't possible as the binary is forced to be x86_64.

## How is the fix applied?

Using the docker automatic TARGETARCH arg to parameterize the download

## What GitHub issue(s) does this PR fix or close?

n/a

### CODE REVIEW

**Review by nrwiersma — APPROVED** [review]
LGTM

File: cmd/proxy/Dockerfile [hunk 1]
```diff
@@ -44,7 +45,8 @@ RUN chmod 644 /config/config.toml
 RUN apk add --update git git-lfs mercurial openssh-client subversion procps fossil tini
 
 # Add git-credential-github-app for native integration with GitHub Apps
-RUN wget -O git-credential-github-app.tar.gz https://github.com/bdellegrazie/git-credential-github-app/releases/download/v0.3.0/git-credential-github-app_v0.3.0_Linux_x86_64.tar.gz \
+RUN if [ "${TARGETARCH}" = "arm64" ]; then ARCH="arm64"; else ARCH="amd64"; fi \
```
Comment (Meroje) [inline cmd/proxy/Dockerfile:48]: @nrwiersma I'm sorry I messed up this one, should've been x86_64 in the second case

Comment (nrwiersma) [reply cmd/proxy/Dockerfile:48]: You can open a new PR, no stress. I should have caught this.

**Comment by dmarkhas** [comment]
Hi @nrwiersma, could you please create a new release (and a new Docker image) that would contain this fix?

**Pull request merged**